.github/workflows/ci.yaml

```
@@ -1,13 +1,19 @@
 name: CI
-on: push
+on:
+  - push
+  - pull_request
 jobs:
   test:
-    runs-on: ubuntu-20.04
     strategy:
+      fail-fast: false
       matrix:
+        os:
+          - ubuntu-20.04
+          - windows-2019
```

Unless we have an issue with a specific OS version, we can go for the `latest` versions:

```suggestion
          - ubuntu-latest
          - windows-latest
```

That way we can notice quickly if a new OS version is not compatible with the project, and we don't have to maintain the versions.